modules/ROOT/pages/intro-platform-hosting.adoc

```diff
@@ -0,0 +1,146 @@
+| API Designer | Y | Y | Y | Y
+| Anypoint Studio | Y | Y | Y | Y
+| Flow Designer | Y | Y | N | N
+| Anypoint Exchange | Y | Y | Y | Y
+| Anypoint API Community Manager* | Y | Y | N | N
+| Anypoint Partner Manager | Y | N | N | N
+| Access management | Y | Y | Y | Y
+| Analytics | Y | Y | N | N
+| Anypoint API Manager | Y | Y | Y | Y
+| Anypoint Runtime Manager | Y | Y | Y | Y
+| Anypoint Monitoring | Y | Y | N | N
+| Secrets manager | Y | Y | N | N
+| Anypoint Visualizer | Y | Y | N | N
+| Anypoint Security edge policies | Y | Y | N | N
+| Anypoint Security tokenization | Y | Y | N | N
+| CloudHub runtimes | Y | Y | Y | N
+| Runtime Fabric | Y | Y | N | N
+| Customer-hosted Runtimes | Y | Y | N | Y
+|===
+
+== Runtime Plane Hosting Options
+
+MuleSoft provides three different options for hosting the Anypoint Platform runtime plane:
+
+* CloudHub
++
+CloudHub is MuleSoft's cloud-based environment for hosting the Mule runtime server and related services. CloudHub enables you to deploy an API or a Mule application on a platform that’s managed by MuleSoft.
++
+CloudHub also provides high-availability, clustering and failover of your APIs and Mule applications and load balances traffic to them. See xref:runtime-manager::cloudhub.adoc[CloudHub] for more information.
++
+CloudHub can only be managed by the Cloud control plane in the same environment (US Cloud, EU Cloud, or GovCloud).
+
+* Runtime Fabric
++
+Runtime Fabric is a container service that enables you to run Mule applications and API gateways within a data center or 3rd-party cloud environment that you control and manage. Runtime Fabric can be installed on a set of physical servers, virtual machines, or within Amazon Web Services and Microsoft Azure.
++
+Runtime Fabric comes bundled with technology such as Docker and Kubernetes to offer benefits such as high-availability, failover, clustering, and load balancing. See xref:runtime-fabric::/latest/index.adoc[Anypoint Runtime Fabric Overview] for more information.
++
+Runtime Fabric can only be managed by Cloud control planes (US Cloud, EU Cloud). MuleSoft Government Cloud and Private Cloud Edition do not support Runtime Fabric.
+
+* Standalone Runtimes
++
+The standalone option enables you to host Mule runtime engine server and related services in an environment that you manage. Using standalone runtimes, the Mule Runtime Server can run on a physical server, a virtual machine, or within a 3rd-party cloud installation like Amazon Web Services or Microsoft Azure.
++
+When using standalone runtimes, you are also responsible for providing the framework for high-availability, failover, clustering and load balancing. See xref:mule-runtime::latest/mule-standalone[Run Mule Runtime Engine On-Premises]
++
+Standalone runtimes can be managed by Cloud control plans (US Cloud, EU Cloud) or a customer-hosted control plane (PCE). MuleSoft Government Cloud does not support standalone runtimes.
```

Comment:
ditto globals:
I just thought of changing this repeated sentence to active voice, so retrofit if you can:
```suggestion
You can manage standalone runtimes by using cloud control plans (US cloud and EU cloud) or a customer-hosted control plane (Anypoint Platform PCE). MuleSoft Government Cloud does not support standalone runtimes.
```

Reply:
Resolved.